front-end/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
We don't really need the boilerplate README that came with "Create React App".

This should be cut down to the few lines that are relevant to what we're doing.